### PR TITLE
Set starting send bitrate to 1000kbps to reduce time of low resolution at start of connection

### PR DESCRIPTION
**Pull request by hensmi-amazon**
**Issue #:** https://github.com/aws/amazon-chime-sdk-component-library-react/issues/968

**Description of changes:** The original behavior isn't necessarily a _bad_ thing, however I think the benefit of starting your video close to the intended quality on a clear network outweighs the risk aversion of a lower start bitrate. On a low-medium RTT connection I did not see any different in loss when enabling video before/after the change with my network constrained.

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
1. Enable video
2. Should immediate be expected resolution

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
y

3. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
n

4. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
n

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sdp/SDP.ts [hunk 1]
```diff
@@ -927,8 +927,9 @@ export default class SDP {
         // `x-google-start-bitrate` is an unofficial flag that has existed in libwebrtc since its release and is unlikely
         // to be removed without notification.
         //
-        // libwebrtc reference: https://webrtc.googlesource.com/src/+/b6ef1a736ee94d97cc28f3bd59b826c716a3278f/media/base/media_constants.cc#97
-        const newLine = line + `;x-google-start-bitrate=${bitrateKbps * 1000}`;
```
Comment (hensmi-amazon) [inline src/sdp/SDP.ts:931]: oopsie daisy

**Review by shi-su — APPROVED** [review]

**Review by shi-su — APPROVED** [review]

File: src/sdp/SDP.ts (hunk 1 above)
Comment (devalevenkatesh) [reply src/sdp/SDP.ts:931]: Wow, wonder how this would have worked so far with * 1000 already added.

Comment (hensmi-amazon) [reply src/sdp/SDP.ts:931]: It just uses the max bitrate configured, so nothing terrible.

**Review by devalevenkatesh — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,32 +5,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [3.26.0] - 2024-10-07
```
Comment (ltrung) [inline CHANGELOG.md:8]: Should still be Unreleased?

**Pull request merged**